components/tabs/tab-mixin.js

```diff
@@ -0,0 +1,193 @@
+import '../colors/colors.js';
+import { css, html, unsafeCSS } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { getFocusPseudoClass } from '../../helpers/focus.js';
+import { SkeletonMixin } from '../skeleton/skeleton-mixin.js';
+
+const keyCodes = {
+	ENTER: 13,
+	SPACE: 32
+};
+
+export const TabMixin = superclass => class extends SkeletonMixin(superclass) {
+	static get properties() {
+		return {
+			selected: { type: Boolean, reflect: true },
+		};
+	}
+
+	static styles = css`
+		:host {
+			box-sizing: border-box;
+			display: inline-block;
+			max-width: 200px;
+			outline: none;
+			position: relative;
+			vertical-align: middle;
+		}
+		.d2l-tab-text {
```

Comment:
We've got this `text`-specific CSS, and other references to the `text` property in render as well. I think we need to make a decision as to whether that belongs here or in the subclass.

Reply:
Think wholesale move `text` specific items to the subclass scope makes sense